src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -893,9 +872,28 @@ private string NaiveXmlDecode(string snippet)
             return snippet.Replace("&amp;", "&").Replace("&lt;", "<").Replace("&gt;", ">").Replace("&quot;", "\"").Replace("&apos;", "'");
         }
 
+        private IList<CodeFlow> CreateCodeFlows(ContrastLogReader.Context context)
+        {
+            return context.PropagationEvents == null ?
+                null :
+                new List<CodeFlow>
+                {
+                    new CodeFlow
+                    {
+                         ThreadFlows = new List<ThreadFlow>
+                         {
+                             new ThreadFlow
+                             {
+                                  Locations = context.PropagationEvents
+                                  // TODO: Populate ImmutableState from the headers.
```

Comment:
>ImmutableState [](start = 52, length = 14)

I was trying to do this when I tripped over the missing properties.

Note how the DRYing out means that every rule that has code flows will benefit from automatically getting the headers.